src/components/table/README.md

```diff
@@ -263,9 +263,9 @@ export default {
 <!-- table-fields-object.vue -->
 ```
 
->**Notes:** 
+**Notes:** 
 - if a `key` property is defined in the field definition, it will take precedence over the key used to define the field.
-- It is possible to define `key` as column's object property, but currently, sorting of these columns is **not supported**
+- You can set the `key` to be a property of a column (e.g. `age.months`), but currently sorting of these columns is **not supported**.  
```

Comment:
Actually this is a bit out of date.

It is now possible (using the built-in sort compare routine) to sort based on dot-notation keys.

It looks like when this feature was added the documentation was not up updated.

https://github.com/bootstrap-vue/bootstrap-vue/blob/dev/src/components/table/table.js#L64-L73

Reply:
Fixed.